.github/workflows/ci.yml

```
@@ -0,0 +1,128 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+    branches:
+      - "master"
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macOS-latest
+          - ubuntu-latest
+        python-version:
+          - 3.6
+          - 3.7
+          - 3.8
+    env:
+      CI_OS: ${{ matrix.os }}
+      PYVER: ${{ matrix.python-version }}
+      PACKAGE: forcebalance
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        activate-environment: test
+        channel-priority: true
+        environment-file: devtools/conda-envs/test_env.yaml
+        auto-activate-base: false
+
+    - name: Additional info about the build
+      shell: bash
+      run: |
+        uname -a
+        df -h
+        ulimit -a
+
+    - name: Environment Information
+      shell: bash -l {0}
+      run: |
+        conda info --all
+        conda list
+
+    - name: Install OpenFF stack and OpenEye on Python 3.6+
+      if: ${{ matrix.python-version != 2.7}}
+      shell: bash -l {0}
+      run: |
+        conda install openforcefield -c conda-forge -c omnia -y
+
+#       Need to replace ndcctools with this block
+#    - name: Install Work Queue
+#      shell: bash -l {0}
+#      run: |
+#        wget https://raw.githubusercontent.com/leeping/forcebalance/master/tools/install-cctools.sh
+#        bash install-cctools.sh
+#        echo "Checking for Work Queue import; if successful, no message will be printed"
+#        python -c "import work_queue"
+#        export PATH="$GITHUB_WORKSPACE/opt/cctools/current/bin:$PATH"
+
+    - name: Install GROMACS
+      shell: bash -l {0}
+      run: |
+        # This will not install double precision, needs to be replaced with a fresh build
```

Since it's easy to lose the context of these sorts of comments
```suggestion
        # TODO: As a result of the move from Travis to GitHub Actions, the CI no longer builds
        # GROMACS from source with double precision. This will need to be reverted so 
        # that the bromine test can return to its original strictness.
```